files/fr/web/api/htmliframeelement/csp/index.html

```diff
@@ -0,0 +1,26 @@
+---
+title: HTMLIFrameElement.csp
+slug: Web/API/HTMLIFrameElement/csp
+browser-compat: api.HTMLIFrameElement.csp
+translation_of: 'HTMLIFrameElement.csp'
+---
+<p>{{SeeCompatTable}}{{APIRef("HTML DOM")}}</p>
+
+<p>La propriété <strong><code>csp</code></strong> de l'interface {{domxref("HTMLIFrameElement")}} spécifie la <a href="/fr/docs/Web/HTTP/CSP">Content Security Policy</a> (ou stratégie de sécurité du contenu) qu'un document embarqué doit accepter pour pouvoir s'auto-appliquer.</p>
```

Comment:
I think the intent of the English wording is closer to "qu'un document embarqué doit accepter de respecter" (cf. example from https://w3c.github.io/webappsec-cspee/#csp-attribute )

Reply:
Ah! Thank you, I had a hard time with that one sentence :) 
Fixed in https://github.com/mdn/translated-content/pull/1790/commits/f4d36cc635776c889b2ba9c37bd2e9743752a4f4